x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_actions/rule_actions.cy.ts

```diff
@@ -74,5 +76,35 @@ describe(
         expect(response.body.hits.hits[0]._source).to.deep.equal(expectedJson);
       });
     });
+
+    it('Allows adding alerts filters for the action', function () {
```

Comment:
Not to block PR: but if there's any way to add this check within a unit test - that might provide us more stability.

Reply:
Done